.github/workflows/client-release-issue.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4.1.2
       - name: Upload tools
-        uses: actions/upload-artifact@v3
```

Comment:
Why `v4` for client release and runtime release issue?

Reply:
I only want to downgrade those that fail.